python/ray/dag/dag_node_operation.py

```diff
@@ -122,14 +153,40 @@ def __hash__(self):
         """
         return hash((self.operation, self.task_idx))
 
+    def __str__(self):
+        class_name = (
```

Comment:
I think if you simply call `str(self.actor_handle)` it will already have the class name and actor ID in it.

Reply:
I'd like to abbreviate the actor_id as well, the full ID is too verbose for visualization.